extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -1188,31 +1186,8 @@ public void handle(AsyncResult<HttpServer> event) {
                                     actualHttpPort = actualPort;
                                     schema = "http";
                                 }
-                                portPropertiesToRestore = new HashMap<>();
-                                String portPropertyValue = String.valueOf(actualPort);
-                                //we always set the .port property, even if we are in test mode, so this will always
-                                //reflect the current port
-                                String portPropertyName = "quarkus." + schema + ".port";
-                                String prevPortPropertyValue = System.setProperty(portPropertyName, portPropertyValue);
-                                if (!Objects.equals(prevPortPropertyValue, portPropertyValue)) {
-                                    portPropertiesToRestore.put(portPropertyName, prevPortPropertyValue);
-                                }
-                                if (launchMode == LaunchMode.TEST) {
-                                    //we also set the test-port property in a test
-                                    String testPropName = "quarkus." + schema + ".test-port";
```

Comment:
I have kept the original behaviour here, but this results in `quarkus.https.test-port` for Vert.x TLS tests. Is this correct, given that the formal property is `quarkus.http.test-ssl-port`?